publish.yml

```diff
@@ -6,7 +6,7 @@ site:
 content:
   sources:
   - url: ./
-    branches: ['3.5','4.0','4.1','4.2','4.3','4.4','HEAD']
+    branches: ['3.5','4.0','4.1','4.2','4.3','4.4','5.x','HEAD']
```

Comment:
```suggestion
    branches: ['3.5','4.0','4.1','4.2','4.3','4.4','5.26','HEAD']
```

Reply:
Are we going to rename the branch?

Reply:
I think these branches are the Github branches, not versions of the manual.